api/policies/v1/policy_types.go

```diff
@@ -152,13 +152,17 @@ type PolicySpec struct {
 	// Default to 10 seconds.
 	// +optional
 	// +kubebuilder:default:=10
+	// +kubebuilder:validation:Minimum=1
+	// +kubebuilder:validation:Maximum=30
 	TimeoutSeconds *int32 `json:"timeoutSeconds,omitempty"`
 
 	// TimeoutEvalSeconds specifies the timeout for the policy evaluation. After
 	// the timeout passes, the policy evaluation call will fail based on the
 	// failure policy.
 	// The timeout value must be between 1 and 30 seconds.
 	// +optional
+	// +kubebuilder:validation:Minimum=1
```

Comment:
I would raise it to be 2 seconds, which is the default value used by Policy Server when nothing is specified by the user.

1 second could be a bit too tight for certain policies

Reply:
Rebased and raised to 2 seconds.